modin/experimental/core/execution/native/implementations/omnisci_on_native/dataframe/partition.py

```diff
@@ -15,7 +15,7 @@
 
 import pandas
 
-from modin.engines.base.frame.partition import PandasFramePartition
+from modin.core.dataframe.pandas.partitioning.partition import PandasFramePartition
```

Comment:
This file should be removed because it is already located in `../partitioning/partition.py`. @devin-petersohn , probably, the file appeared here when merging with master branch.